toplevel/toploop.mli

```diff
@@ -203,3 +203,7 @@ val split_path : string -> string list
     double-quoted (which allows semicolons in filenames to be quoted). The
     double-quote characters are stripped (i.e. [f"o"o = foo]; also
     [split_path "foo\";\";bar" = ["foo;"; "bar"]) *)
+
+val preload_objects : string list ref
+
+val prepare : Format.formatter -> bool
```

Comment:
Could we have some documentation comments on what are `preload_objects` and what is being prepared?

Reply:
I added some, thanks!